taxonomy: Translate INCI functions (#11559)
openfoodfacts/openfoodfacts-server: taxonomies/beauty/inci_functions.txt

description:sv: Skapar tryck i aerosolförpackningar och driver ut innehållet när ventilen öppnas. Vissa drivmedel i vätskeform kan fungera som lösningsmedel.

en: Antioxidant
ar: مضاد للأكسدة
bg: Антиоксидант
cs: Antioxidant
da: Antioxidant
de: Antioxidationsmittel
el: Αντιοξειδωτικό
es: Antioxidante
et: Antioksüdant
fi: Antioksidantti
fr: Antioxydant
he: נוגד חמצון
hi: एंटीऑक्सीडेंट
hu: Antioxidáns
id: Antioksidan
it: Antiossidante
ja: 抗酸化剤
ko: 항산화제
lt: Antioksidatorius
lv: Antioksidants

> pl: Przeciwutleniająca

Reviewer: why to remove it?

> pl: Przeciwutleniacz

Author: ```suggestion
pl: Przeciwutleniająca, Przeciwutleniacz
```

sk: Depilačná látka
sl: Depilacijsko sredstvo
description:en: Removes unwanted body hair.
description:ar: يزيل شعر الجسم غير المرغوب فيه.
description:bg: Премахва нежеланото окосмяване по тялото.
description:cs: Odstraňuje nežádoucí chloupky na těle.
description:da: Fjerner uønsket kropsbehåring.
description:de: Entfernt unerwünschte Körperhaare.
description:el: Καθιστούν την επιδερμίδα πιο απαλή και λεία.
description:es: Elimina el vello corporal no deseado.
description:et: Eemaldab soovimatu kehakarva.

> description:fi: Poistaa ei-toivotut ihokarvat.

Author: ```suggestion
description:fi: Vähentää tai peittää epämiellyttäviä kehon hajuja. Poistaa ei-toivotut ihokarvat.
```

description:fr: Supprime les pilosités indésirables.
description:he: מסיר שיער גוף לא רצוי.
description:hi: अनचाहे शरीर के बालों को हटाता है।

> description:hu: Eltávolítja a nem kívánt testszőrzetet.

Author: ```suggestion
description:hu: Puhává és simává teszi a bőrt. Eltávolítja a nem kívánt testszőrzetet.
```

sl: Sredstvo za nego nohtov
sv: Nagelkonditionerande
description:en: Improves the cosmetic characteristics of the nail.
description:ar: يحسن الخصائص التجميلية للأظافر.
description:bg: Подобрява козметичните характеристики на ноктите.
description:cs: Zlepšuje kosmetické vlastnosti nehtů.
description:da: Forbedrer neglens kosmetiske egenskaber.
description:de: Verbessert die kosmetischen Eigenschaften der Nägel.
description:el: Βελτιώνει τα καλλυντικά χαρακτηριστικά των νυχιών.
description:es: Mejora las características cosméticas de las uñas.

> description:et: Parandab küünte kosmeetilisi omadusi.
> description:fi: Parantaa kynsien kosmeettisia ominaisuuksia.

Author: ```suggestion
description:et: Annab kosmeetikavahendile küütleva värvitooni.
description:fi: Tuo iholle miellyttävää raikkautta.
```

description:fr: Améliore les caractéristiques cosmétiques des ongles.
description:he: משפר את המאפיינים הקוסמטיים של הציפורניים.
description:hi: नाखून के सौंदर्य प्रसाधन विशेषताओं में सुधार करता है।
description:hu: Javítja a körmök kozmetikai jellemzőit.
description:id: Meningkatkan karakteristik kosmetik kuku.
description:it: Migliora le caratteristiche cosmetiche delle unghie.
description:ja: 爪の косметиックな特性を改善します。
description:ko: 손톱의 화장품 특성을 향상시킵니다.
description:lt: Gerina kosmetines nagų savybes.

> description:lv: Uzlabo nagu kosmētiskās īpašības.
> description:nl: Verbetert de cosmetische eigenschappen van de nagel.
> description:pl: Poprawia kosmetyczne właściwości paznokci.

Author: ```suggestion
description:lv: Palīdz aizsargāties pret iekaisumu.
description:nl: Verbetert de oplosbaarheid van een stof die slecht oplosbaar is in water.
description:pl: Poprawia charakterystykę kosmetyczną paznokcia.
```
